scripts/tag-release-candidate.sh

```diff
@@ -79,9 +80,10 @@ if [ -z "$CS_VERSION" ]; then
   exit 1
 fi
 
-# If a user omits the leading "v" (enters 1.2.3 instead of v1.2.3), handle it.
-if ! [[ "${CS_VERSION}" = v* ]]; then
-  CS_VERSION="v${CS_VERSION}"
```

Comment:
Why not keeping this?

Reply:
IMO having a stricter check is preferred, as it leaves less room for error. The `__build` make target also builds on the assumption of the well formed version for finding the name of the RC tag.

The plan is for this to all be run via automation, so it shouldn't be a large inconvenience users.